web/src/app/users/UserDetails.tsx

```diff
@@ -73,6 +73,15 @@ const profileQuery = gql`
   }
 `
 
+export function useSuspenseContext(): Partial<OperationContext> {
```

Comment:
I added this here at the top level `UserDetails` because `web/src/users/util.js` is not yet in TypeScript